terraform/kubernetes-cluster/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "delta_cert_expiration_seconds" {
+  description = "Number of seconds until the Kubernetes certificate expires"
+  default     = 31536000 # 1 year
+}
```

Comment:
See this ![image](https://github.com/franciscoprin/kubernetes-the-hard-way-aws-terraform/assets/11096782/296b94ee-d53b-484d-8af9-c81f13bf372f)?

That means that you can't go here via 🚗 
Or set .editor-config/pre-commit hooks :)

Reply:
It should be set here: https://github.com/franciscoprin/kubernetes-the-hard-way-aws-terraform/pull/1/commits/48e60d50e554bac5741c838949c464bf0cf40635, thanks so much for this advice :) .

Reply:
Any time. Feel free to check our infra repo settings for these files - you could find something else useful for your case